app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -207,6 +210,7 @@ class PinPasswordActivityTest {
   }
 
   @Test
+  @DisableAccessibilityChecks
```

Comment:
For this & the other checks, can you provide some context as to why these need to be exempted?

Reply:
First I have run all expresso test without DisableAccessibilityChecks and after seeing the error (which is related to accessibility) I have added it.

Reply:
> @BenHenning  First I have run all expresso test without DisableAccessibilityChecks and after seeing the error (which is related to accessibility) I have added DisableAccessibilityChecks annotation.

Reply:
This turns off the checks for accessibility, though, which we generally only ever want to do on a short-term and conditional basis (we shouldn't be broadly disabling accessibility checks without a clear need, and we should actively be trying to remove those disables since it means we aren't verifying accessibility correctness on those screens). It's not clear to me why these are correct to disable--it actually seems like we may want to wait to add OppiaTestRule to this & other files until we can fix the accessibility test issues in for those screens.